src/java.base/share/classes/java/lang/System.java

```diff
@@ -2130,6 +2130,8 @@ private static Properties createProperties(Map<String, String> initialProps) {
         return properties;
     }
 
+    private static InputStream initIn = null;
```

Comment:
I think it would be helpful to move this to be with `in` and `out`, and also to add a comment to say that it's the initial value of `in`, set in initPhase1.  It doesn't need to be initialized to null as that it's default value anyway.